.github/commands.yml

```diff
@@ -0,0 +1,12 @@
+{
+	perform: true,
+	commands: [
+        {
+            type: 'label',
+            name: 'duplicate',
+            allowTriggerByBot: true,
+            action: 'close',
+            comment: "Thanks for creating this issue! We figured it's covering the same as another one we already have. Thus, we closed this one as a duplicate. You can search for existing issues [here](https://aka.ms/vscodeissuesearch). See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.\n\nHappy Coding!"
```

Comment:
the links in this message need to be updated

Reply:
Yep, @yualan is working on that now.